roles/st2/tasks/main.yml

```diff
@@ -1,22 +1,6 @@
 ---
 
-- name: Install latest st2 package
-  become: yes
-  apt:
-    name: st2
-    state: latest
-  when: st2_version == "stable"
-  notify:
-    - restart st2
-
-- name: Install latest st2 package
-  become: yes
-  apt:
-    name: st2={{ st2_version }}-{{ st2_revision }}
-    state: present
-  when: st2_version != "stable"
-  notify:
-    - restart st2
+- include: "{{ ansible_os_family }}.yml"
```

Comment:
Done in https://github.com/StackStorm/ansible-st2/pull/66/commits/3c1fd1436946371d73a8ada257492b21d7e54995